src/integrations/prefect-dbt/tests/cli/test_commands.py

```diff
@@ -561,7 +561,7 @@ def test_append_dirs_to_commands(
 async def test_run_dbt_build_creates_artifact(profiles_dir, dbt_cli_profile_bare):
     @flow
     async def test_flow():
-        return await run_dbt_build(
+        return run_dbt_build(
```

Comment:
I wouldn't expect these tests to pass since `sync_compatible` functions should run as async in async flows. I think the order of the decorators needs to be reversed.